Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/signposts/setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup, Extension
+
+module = Extension(
+    'signposts',
+    sources=['signposts.c'],
+)
+
+setup(
+    name='signposts',
+    version='1.0.0',
+    description='provides python api for emitting os_signposts while running a browser benchmark',
```

Comment:
Does this require command line tools installed at least to get it working? Should we put it into autoinstalled?
And should we include this step if user trying to run on a browser that does not support signpost?